tests/unit/unit_tests_common.c

```diff
@@ -12,71 +12,30 @@
 #include "unit_tests.h"
 
 /*
- * Setup function to manage output log streams from most (unit) tests.
- * By default, there will be no output. Info / error messages are only printed
- * when the caller sets the VERBOSE env var to opt-in.
- *
- * NOTE about exp_msg_level in set_log_streams_for_*tests() functions:
- *
- * A unit-test, or a test case, that invokes these interfaces is declaring that
- * messages logged when running this test/case will be visible only when the
- * message verbosity is >= than this expected message level. Therefore, to see
- * these messages, the test-runner will have to invoke the test/case with a
- * verbosity level >= expected message level. Otherwise, the output log handles
- * will be set to /dev/null which will suppress all test output.
- */
-void
-set_log_streams_for_tests(msg_level exp_msg_level)
-{
-   if (Ctest_verbosity >= exp_msg_level) {
-      platform_set_log_streams(stdout, stderr);
-   } else {
-      platform_reset_log_streams();
-   }
-}
-
-/*
- * Setup function is provided to manage output log streams from (unit) tests
- * that are induce error conditions and check that error messages are raised
- * as expected.
+ * Setup function is provided to manage output log streams from (unit) tests.
+ * By default, execution from unit-tests will be silent, redirecting output
+ * handles to /dev/null. Info / error messages are only printed when the caller
+ * sets the VERBOSE env var to opt-in.
  *
  * Some unit tests exercise error cases, so even when everything succeeds, they
  * generate lots of "error" messages. By default, that would go to stderr, which
  * would pollute the test output. This function sets up output file handles such
  * that those expected error messages are only printed when the caller sets the
- * VERBOSE env var to opt-in. By default, execution from unit-tests will be
- * silent, redirecting output handles to /dev/null.
- *
- * Returns: Output file handles (out/error) if caller has requested for them.
+ * VERBOSE env var to opt-in.
```

Comment:
You may want to review this edited prolog comment block ... for accuracy.

Reply:
I think it is accurate, but perhaps incomplete.  It would be good to call out some actionable instructions for unit test authors.  For example, we could add something like:
> Tests which exercise error paths in the library should first call this function with MSG_LEVEL_ERRORS, so that any error messages generated by the library are only shown when the test is run with VERBOSE >= 3.
>
> Tests which cause the library to generate debug output (e.g. by printing internal data structures) should call this function with MSG_LEVEL_DEBUG, so that that output is only shown when the test is run with VERBOSE>=7.